Apps/Admin/Client/Pages/SupportPage.razor

```diff
@@ -50,58 +50,65 @@
 
 @if (SupportUsersLoaded || SupportUsersLoading)
 {
-    <MudTable Class="mt-3"
-              Items="SupportUserRows"
-              Loading="SupportUsersLoading"
-              AllowUnsorted="false"
-              Breakpoint="Breakpoint.Md"
-              HorizontalScrollbar="true"
-              Striped="true"
-              Dense="true"
-              data-testid="user-table">
-        <HeaderContent>
-            <MudTh>
-                <MudTableSortLabel SortBy="new Func<SupportUserRow, object>(x => x.Hdid)">
-                    HDID
-                </MudTableSortLabel>
-            </MudTh>
-            <MudTh>
-                <MudTableSortLabel SortBy="new Func<SupportUserRow, object>(x => x.PersonalHealthNumber)">
-                    PHN
-                </MudTableSortLabel>
-            </MudTh>
-            <MudTh>
-                <MudTableSortLabel SortBy="new Func<SupportUserRow, object>(x => x.LastLoginDateTime)"
-                                   InitialDirection="@SortDirection.Descending">
-                    Last Login
-                </MudTableSortLabel>
-            </MudTh>
-            <MudTh Style="text-align:right"></MudTh>
-        </HeaderContent>
-        <RowTemplate>
-            <MudTd DataLabel="HDID" data-testid=@($"user-table-hdid-{context.Hdid}")>@context.Hdid</MudTd>
-            <MudTd DataLabel="PHN" data-testid=@($"user-table-phn-{context.Hdid}")>@context.PersonalHealthNumber</MudTd>
-            <MudTd DataLabel="Last Login" data-testid=@($"user-table-last-login-{context.Hdid}")>@DateFormatter.ToShortDateAndTime(context.LastLoginDateTime)</MudTd>
-            <MudTd Style="text-align:right">
-                <div>
-                    <MudTooltip Text="@(context.IsExpanded ? "Collapse" : "Expand")">
-                        <MudIconButton OnClick="@(() => ToggleExpandRow(context.Hdid))"
-                                       Icon="@(context.IsExpanded ? Icons.Material.Filled.ExpandLess : Icons.Material.Filled.ExpandMore)"
-                                       Color="@Color.Primary"
-                                       data-testid=@($"messaging-verification-table-expand-btn-{context.Hdid}")>
-                        </MudIconButton>
-                    </MudTooltip>
-                </div>
-            </MudTd>
-        </RowTemplate>
-        <ChildRowContent>
-            @if (context.IsExpanded)
-            {
-                <MessageVerificationTable Data="@GetMessagingVerificationModels(context.Hdid)" IsLoading="@MessagingVerificationsLoading" />
-            }
-        </ChildRowContent>
-        <PagerContent>
-            <MudTablePager />
-        </PagerContent>
-    </MudTable>
+    <MudGrid>
+        <MudItem xs="12" lg="12">
+            <MudText Class="mt-3" Typo="Typo.subtitle1">
+                <strong>@SelectedQueryType Query Results</strong>
```

Comment:
`@SelectedQueryType` should be swapped out for `@FormatQueryType(SelectedQueryType)` to achieve the proper casing.

Reply:
fixed